README.txt

```diff
@@ -11,10 +11,10 @@ qserv-check-integration.py --case=01 --load --mode=qserv
 # to run integration tests for one case, only for mysql
 qserv-check-integration.py --case=01 --load --mode=mysql
 
-# to run multi-node, use -M
-qserv-check-integration.py --case=01 --load -M --mode=qserv
+# to run multi-node tests for one case, use -M
+qserv-check-integration.py --case=01 --load -M
```

Comment:
`qserv-test-integration.py` and `qserv-check-integration.py` are separate scripts, with the latter allowing a lot more control over individual options. I left the specific option to run in multi-node setup there for that purpose, but if we want to do it "automatically" for check-integration too by reading `node_type` in the config files I can do that.